### DISCUSSION TITLE
How to remove a message from the remote queue?

### DISCUSSION THREAD

**Discussion by zb226** [emersion/maddy#582] [Q&A] [answered]
I'm trying out Maddy and got into situations where my mail would not be accepted by the remote host, for example due to erroneous DNS configuration on my end. Maddy will then schedule to retry:

`queue: will retry {"attempts_count":{"...":1},"msg_id":"94c089b9","next_try_delay":"14m59.999999207s","rcpts":["..."]}`

Needing time to fix the DNS, I wanted to clear that message from the queue so it doesn't get resent. When I deleted the message files from `/var/lib/maddy/remote_queue` it resulted in an error:

`queue: read message {"reason":"open /var/lib/maddy/remote_queue/94c089b9.meta: no such file or directory"}`

That doesn't seem to hurt but I asked myself if there is a "clean" way to do it?

**Comment by foxcpp** [comment, marked as answer]
>That doesn't seem to hurt but I asked myself if there is a "clean" way to do it?

No, maddy will just discard queue entries if it fails to read the `.meta` file so it is kinda clean already.

**Reply by zb226** [reply]
👍
